ui-v2/app/helpers/format-number.js

```diff
@@ -0,0 +1,19 @@
+import { helper } from '@ember/component/helper';
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
+// the actual helper
+const toLocaleString = function(num) {
+  return num.toLocaleString();
```

Comment:
Oh, I see now why you wrote the code above. Your number formatting doesn't concern itself with precision. 

Personally, I think it _should_ concern itself with precision. It's the easiest way to make that sort of thing consistent across the code. 

+1 to using native `toLocaleString` as well though.

Reply:
Completely agree here. In this case I only need 2 decimal places for 'view' reasons.

The reasons I didn't add the precision to the helper itself was because it wasn't that straightforwards (see below), plus I only needed it for the tomography bits, which is pretty much lifted from the old UI anyway, and ideally the whole thing needs it's concerns addressing (it includes a 'size' setting also).

I did look at using the `minimumFractionDigits` setting of `toLocaleString`/`Intl`

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Parameters
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat#Parameters

But Android support was unclear (seems to be fine on Chrome but nothing else??), and as this began as a simple 'time to upgrade some addons' I took the safer approach.

If you know that we are safe using the `minimumFractionDigits` setting, then I can fix it up to use that (which would be ideal) - if not then I can add something custom for specifying precision (maybe move the above code into the helper and include a `fixed=` param?) or go back to using the addon I was using (it had a `format="0,000.00"` way of asking how you wanted it to be formatted)

Whichever, I'd probably do that in a separate PR if you think thats a good plan?

Reply:
I did a separate PR for passing through the `options`, see #4703 

Happy to either close or merge that one ontop of here.